docs/blog/author.yaml

```diff
@@ -339,3 +339,7 @@
 - id: Grant Glidewell
   bio: New dad moving modern technologies forward with cutting-edge projects @ThirdAndGrove. And doggos
   avatar: avatars/grant-glidewell.jpg
+- id: Ilesh Mistry
+  bio: https://www.ileshmistry.com | Gatsby and GraphQL fan | @kentico MVP working at @MMT_Digital uses KenticoCloud | @ManUtd fan | Husband | Father | Traveller | @Marvel fan
+  avatar: avatars/ilesh-mistry.jpg
```

Comment:
This path doesn't match the same capitalization as the image:

`avatars/ilseh-mistry.jpg` vs `avatars/Ilesh-Mistry.jpg`

This might not break locally on some environments, but I fear it could be hard to catch when the site builds and is deployed to another environment. To match the convention of other images in this folder could you rename the file to match the kebab-case/lowercase hyphenated ones?

Reply:
Ah okay, so originally I copied over a file Ilesh-Mistry, but renamed it, however it looks like the rename part to lower case didn't come through. 

I will retry this, thanks.